rediscluster/cluster.go

```diff
@@ -95,8 +95,12 @@ type Opts struct {
 	RoundRobinSeed RoundRobinSeed
 	// LatencyOrientedRR - when MasterAndSlaves is used, prefer hosts with lower latency
 	LatencyOrientedRR bool
-	TLSEnabled        bool
-	TLSConfig         *tls.Config
+	// Enable connection with TLS
+	TLSEnabled bool
+	// Config for TLS connection
+	TLSConfig *tls.Config
+	// Do not resolve host address to ip
+	SkipHostResolving bool
```

Comment:
It looks like there is no need in `SkipHostResolving`.

Reply:
`SkipHostResolving` removed